### PR TITLE
Pulls/4.0/requirements module resource port

### PR DESCRIPTION
**Pull request by unclecheese**
Fixes https://github.com/silverstripe/silverstripe-framework/issues/7130

### CODE REVIEW

**Comment by dhensby** [comment]
code style issue fixed on merge

**Comment by unclecheese** [comment]
Was this PR ever approved?

**Comment by dhensby** [comment]
It was by me :)

**Comment by sminnee** [comment]
I think that that this regex would benefit from a ^ and $

File: src/View/Requirements_Backend.php [hunk 1]
```diff
@@ -643,7 +643,7 @@ public function css($file, $media = null)
     protected function parseModuleResourceReference($file)
     {
         // String of the form vendor/package:resource. Excludes "http://bla" as that's an absolute URL
-        if (preg_match('#([^ ]*/[^ ]*) *: *([^ ]*)#', $file, $matches)) {
+        if (preg_match('#([^\/\/][^ /]*\/[^ /]*) *: *([^ ]*)#', $file, $matches)) {
```
Comment (tractorcow) [inline src/View/Requirements_Backend.php]: Do `[^\/\/]` and `[^\/]` differ at all? It's the same character repeated twice in a char list.

**Pull request merged**